packages/amplify-category-auth/src/provider-utils/awscloudformation/index.js

```diff
@@ -217,8 +217,15 @@ function getOAuthProviderKeys(currentEnvSpecificValues, resourceParams) {
   const configuredProviders = JSON.parse(hostedUIProviderCreds).map(h => h.ProviderName);
   const deltaProviders = _.intersection(oAuthProviders, configuredProviders);
   deltaProviders.forEach(d => {
-    currentEnvSpecificValues[`${d.toLowerCase()}AppIdUserPool`] = configuredProviders[`${d.toLowerCase()}AppIdUserPool`];
-    currentEnvSpecificValues[`${d.toLowerCase()}AppSecretUserPool`] = configuredProviders[`${d.toLowerCase()}AppSecretUserPool`];
+    if (d === 'SignInWithApple') {
+      currentEnvSpecificValues[`${d.toLowerCase()}ClientIdUserPool`] = configuredProviders[`${d.toLowerCase()}ClientIdUserPool`];
```

Comment:
There are a lot of `d.toLowerCase()` instances here. Can they be put in a variable to make the code more readable.

Reply:
updated